lib/svg-sprite.js

```diff
@@ -246,7 +246,7 @@ SVGSpriter.prototype._compile = function() {
         const [config, cb] = this._compileQueue.shift();
 
         // If this is a modeless run
-        if (config === {}) {
+        if (Object.keys(config).length === 0) {
```

Comment:
Here was the error. Yes, this branch was never executed before. And the tests was kind of exploiting this bug. 
I changed the test, but TBH this should be in another PR.

Reply:
But this is now flagged by ESLint here so I'm not sure. I will probably rebase the PR and just split the patches before the merge.

Thanks for fixing the issue!